x11-misc/revelation/metadata.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <!-- maintainer-needed -->
```

Comment:
We **can't** add new packages without dedicated maintainers. 
https://projects.gentoo.org/qa/policy-guide/maintainer.html#pg0602 (New packages without maintainers)
Besides this package was removed because it didn't have a maintainer.

You can add yourself + proxy-maint project as described in here,
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml